src/Microsoft.VisualStudio.AppDesigner/Resources/Designer.Designer.vb

```diff
@@ -14,56 +14,62 @@ Option Explicit On
 Imports System
 
 Namespace My.Resources
-
+    
     'This class was auto-generated by the StronglyTypedResourceBuilder
     'class via a tool like ResGen or Visual Studio.
     'To add or remove a member, edit your .ResX file then rerun ResGen
     'with the /str option, or rebuild your VS project.
     '''<summary>
     '''  A strongly-typed resource class, for looking up localized strings, etc.
     '''</summary>
-    <Global.System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "16.0.0.0"),
-     Global.System.Diagnostics.DebuggerNonUserCodeAttribute(),
-     Global.System.Runtime.CompilerServices.CompilerGeneratedAttribute()>
+    <Global.System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "17.0.0.0"),  _
+     Global.System.Diagnostics.DebuggerNonUserCodeAttribute(),  _
+     Global.System.Runtime.CompilerServices.CompilerGeneratedAttribute()>  _
     Friend Class Designer
-
+        
         Private Shared resourceMan As Global.System.Resources.ResourceManager
-
+        
         Private Shared resourceCulture As Global.System.Globalization.CultureInfo
-
-        <Global.System.Diagnostics.CodeAnalysis.SuppressMessageAttribute("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode")>
+        
+        <Global.System.Diagnostics.CodeAnalysis.SuppressMessageAttribute("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode")>  _
         Friend Sub New()
             MyBase.New
         End Sub
-
+        
         '''<summary>
         '''  Returns the cached ResourceManager instance used by this class.
         '''</summary>
-        <Global.System.ComponentModel.EditorBrowsableAttribute(Global.System.ComponentModel.EditorBrowsableState.Advanced)>
+        <Global.System.ComponentModel.EditorBrowsableAttribute(Global.System.ComponentModel.EditorBrowsableState.Advanced)>  _
         Friend Shared ReadOnly Property ResourceManager() As Global.System.Resources.ResourceManager
             Get
                 If Object.ReferenceEquals(resourceMan, Nothing) Then
+                    ' NOTICE: If this file is regenerated, the value below will change to "Designer". You need to manually change it back to "Microsoft.VisualStudio.AppDesigner.Designer" before pushing the changes.
+                    ' You should also add this comment back to the file so the next person is aware of the issue.
+                    ' Related links:
+                    ' - https://github.com/dotnet/project-system/issues/1058
+                    ' - https://github.com/dotnet/project-system/pull/6202
+                    ' - https://github.com/dotnet/project-system/pull/3741
                     Dim temp As Global.System.Resources.ResourceManager = New Global.System.Resources.ResourceManager("Microsoft.VisualStudio.AppDesigner.Designer", GetType(Designer).Assembly)
```

Comment:
This file was regenerated and I added this big comment linking to the related issues for the problem that happens here when the file is regenerated.